src/openzaak/components/besluiten/tests/test_besluit_closed_zaak.py

```diff
@@ -0,0 +1,149 @@
+            applicatie=self.applicatie,
+            component=ComponentTypes.brc,
+            scopes=[SCOPE_BESLUITEN_AANMAKEN],
+            zaaktype=zaaktype if zaaktype else "",
+            besluittype=self.besluittype_url if besluittype is None else besluittype,
+            max_vertrouwelijkheidaanduiding=self.max_vertrouwelijkheidaanduiding,
+        )
+
+    def _add_zaken_auth(self, zaaktype=None, scopes=None):
+        if scopes is None:
+            scopes = []
+
+        Autorisatie.objects.create(
+            applicatie=self.applicatie,
+            component=ComponentTypes.zrc,
+            scopes=scopes,
+            zaaktype=zaaktype if zaaktype else "",
+            max_vertrouwelijkheidaanduiding=self.max_vertrouwelijkheidaanduiding,
+        )
+
+    def setUp(self):
+        super().setUp()
+
+        self.content = {
+            "verantwoordelijke_organisatie": "517439943",
+            "identificatie": "123123",
+            "besluittype": self.besluittype_url,
+            "datum": "2018-09-06",
+            "toelichting": "Vergunning verleend.",
+            "ingangsdatum": "2018-10-01",
+            "vervaldatum": "2018-11-01",
+            "vervalreden": VervalRedenen.tijdelijk,
+        }
+
+    def test_register_with_closed_zaak(self):
+        self._add_besluiten_auth()
+
+        zaak = ZaakFactory.create(
+            einddatum=timezone.now(), zaaktype__catalogus=self.besluittype.catalogus
+        )
+        zaak_url = reverse(zaak)
+        self.besluittype.zaaktypen.add(zaak.zaaktype)
+
+        self.assertTrue(zaak.is_closed)
+
+        content = self.content.copy()
+        content["zaak"] = f"http://testserver{zaak_url}"
+
+        response = self.client.post(self.url, content)
+
+        self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN, response.data)
+        self.assertEqual(
+            response.data["detail"],
+            _("Je mag geen gegevens aanpassen van een gesloten zaak."),
+        )
+
+    def test_register_with_closed_zaak_with_force_scope(self):
+        zaak = ZaakFactory.create(
+            einddatum=timezone.now(), zaaktype__catalogus=self.besluittype.catalogus
+        )
+        zaak_url = reverse(zaak)
+        self.besluittype.zaaktypen.add(zaak.zaaktype)
+
+        self.assertTrue(zaak.is_closed)
+
+        self._add_besluiten_auth(
+            besluittype=self.besluittype_url,
+            zaaktype=f"http://testserver{reverse(zaak.zaaktype)}",
+        )
+
+        self._add_zaken_auth(
+            scopes=[SCOPE_ZAKEN_GEFORCEERD_BIJWERKEN],
+            zaaktype=f"http://testserver{reverse(zaak.zaaktype)}",
+        )
+
+        content = self.content.copy()
+        content["zaak"] = f"http://testserver{zaak_url}"
+
+        response = self.client.post(self.url, content)
+
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED, response.data)
+
+    def test_register_with_closed_zaak_with_force_scope_on_catalogus(self):
```

Comment:
For this test you'll have to create a `CatalogusAutorisatie` with the geforceerd-bijwerken scope instead to properly test this behavior